app-modules/notification/tests/Tenant/Filament/Actions/SubscribeBulkActionTest.php

```diff
@@ -0,0 +1,181 @@
+      the features or functionality of the software.
+    - You may not move, change, disable, or circumvent the license key functionality
+      in the software, and you may not remove or obscure any functionality in the
+      software that is protected by the license key.
+    - You may not alter, remove, or obscure any licensing, copyright, or other notices
+      of the licensor in the software. Any use of the licensor’s trademarks is subject
+      to applicable law.
+    - Canyon GBS LLC respects the intellectual property rights of others and expects the
+      same in return. Canyon GBS™ and Advising App™ are registered trademarks of
+      Canyon GBS LLC, and we are committed to enforcing and protecting our trademarks
+      vigorously.
+    - The software solution, including services, infrastructure, and code, is offered as a
+      Software as a Service (SaaS) by Canyon GBS LLC.
+    - Use of this software implies agreement to the license terms and conditions as stated
+      in the Elastic License 2.0.
+
+    For more information or inquiries please visit our website at
+    https://www.canyongbs.com or contact us via email at legal@canyongbs.com.
+
+</COPYRIGHT>
+*/
+
+use AdvisingApp\Notification\Models\Subscription;
+use AdvisingApp\Prospect\Filament\Resources\ProspectResource\Pages\ListProspects;
+use AdvisingApp\Prospect\Models\Prospect;
+use AdvisingApp\StudentDataModel\Filament\Resources\StudentResource\Pages\ListStudents;
+use AdvisingApp\StudentDataModel\Models\Student;
+use App\Models\User;
+
+use function Pest\Laravel\actingAs;
+use function Pest\Livewire\livewire;
+
+it('can bulk subscribe students without removing the prior subscriptions', function () {
+    $user = User::factory()->licensed(Student::getLicenseType())->create();
+    $anotherUser = User::factory()->licensed(Student::getLicenseType())->create();
+
+    $user->givePermissionTo('student.view-any');
+    $user->givePermissionTo('student.create');
+    $user->givePermissionTo('student.*.update');
+
+    actingAs($user);
+
+    $students = Student::factory()->has(
+        Subscription::factory()->state([
+            'user_id' => $user->getKey(),
+        ]),
+        'subscriptions'
+    )->count(5)->create();
+
+    $students->each(function (Student $student) use ($user) {
+        expect($student->subscriptions()->where('user_id', $user->getKey())->exists())->toBeTrue();
+    });
+
+    livewire(ListStudents::class)
+        ->callTableBulkAction('bulkSubscription', $students, [
+            'user_ids' => [$anotherUser->getKey()],
+            'remove_prior' => false,
+        ])
+        ->assertSuccessful();
+
+    $students->each(function (Student $student) use ($user, $anotherUser) {
+        expect($student->subscriptions()->where('user_id', $anotherUser->getKey())->exists())->toBeTrue();
+        expect($student->subscriptions()->where('user_id', $user->getKey())->exists())->toBeTrue();
+    });
+})->only();
```

Comment:
`->only()` has been left on this test. Merging this in would result in ALL other tests to not run and this to be the only thing in CI that ran.

Please makes sure to meticulously review your own code before requesting review to catch things like this.